synapseutils/describe_functions.py

```diff
@@ -0,0 +1,81 @@
+from collections import defaultdict
+from json import dumps
+from os import path
```

Comment:
```suggestion
import json
import os
```

Reply:
The rest of the client doesn't import particular functions, so I'm going to stick with that paradigm